ramls/types/proxy.json

```diff
@@ -16,5 +16,8 @@
       "description": "Whether this proxy is inherited",
       "example": true
     }
-  }
-}
\ No newline at end of file
+  },
+  "required": [
```

Comment:
I created a separate proxy.json and proxyPut.json because we do not want user to be able to update "inherited". In mod-kb-ebsco, since UI passes it, we receive it and always set it to false before making a request to RM API. I thought in Java, we can just ignore the property "proxy.inherited" even if passed from UI. Is that possible?